Plugins/Gerrit/FormGerritPublish.cs

```diff
@@ -95,6 +101,13 @@ private bool PublishChange(IWin32Window owner)
             pushCommand.CommandText = PushCmd(
                 _NO_TRANSLATE_Remotes.Text,
                 targetBranch);
+
+            string topic = _NO_TRANSLATE_Topic.Text.Trim();
+            if (!string.IsNullOrEmpty(topic))
+            {
+                pushCommand.CommandText += " -o topic=" + topic;
```

Comment:
Note: This will break if using Git version < 2.10.2. Not supported, but users occasionally tries using older versions.

Reply:
Since we do not force the minimum git version (though I think we should), it is better to wrap this in a git version check.

Reply:
Hi @gerhardol, I have added the reference to the documentation and reasoning for the changes, please let me know if that is sufficient. As of using `-o` flag, I will fix it with git check, as there is still another way to post the same data to Gerrit.

Do you have any documentation or example on how should I proceed with translations? I checked the wiki on GitHub, but failed to find anything.

Reply:
I will approve also with the minor problem with using unsupported Git - if you use latest Gerrit, it is more likely that you use recent Git too. (Also GitArgumentBuilder could be used.)
To merge, I also want at least another Gerrit user to review - I have no feeling for when this should be merged. Also, some other member need to approve.